src/Command.js

```diff
@@ -19,6 +19,47 @@ export default class Command {
     this.concurrency = (!flags || flags.concurrency === undefined) ? DEFAULT_CONCURRENCY : Math.max(1, +flags.concurrency || DEFAULT_CONCURRENCY);
   }
 
+  get name() {
+    // For a class named "FooCommand" this returns "foo".
+    return this.className.replace("Command", "").toLowerCase();
+  }
+
+  get className() {
+    return this.constructor.name;
```

Comment:
Okay, I've got this queued up for a follow-up PR, since it works well as a standalone.

Also added an error on:

- A class that doesn't extend `Command`

Will submit after this lands.